src/main/java/com/lgtm/easymoney/models/User.java

```diff
@@ -7,17 +7,17 @@
 import java.util.Set;
 import javax.persistence.*;
 import javax.validation.constraints.Email;
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+
+import lombok.*;
 
 @Entity
 @Table(
         name="user",
         uniqueConstraints=@UniqueConstraint(name= DBConsts.USER_EMAIL_CONSTRAINT, columnNames={"email"})
 )
 @AllArgsConstructor
-@Data
+@Getter
```

Comment:
Can't use `@Data` as it produced stackoverflow error

Reply:
i'll take a look into this

Reply:
`@Data` includes `toString()` which can cause circular deserialization issue